docs/modules/ROOT/pages/how-tos/appcat/qppuio-quotas.adoc

```diff
@@ -0,0 +1,35 @@
+= Change AppCat service quota on APPUiO Cloud
+
+It's important, that the service instance has been created with resources lower than the quota.
+This is due to the fact, that the quotas are bound to the namespace and in order to change them, we need the namespace itself.
```

Comment:
This is confusing... why the namespace quote is not automatically scaled based on the service instance? If the customer can change the quote of the namespace then this should happen automatically at service creation.

Reply:
The customer can't change the quotas.

This is our internal documentation for increasing the quotas when/if a customer asks us to do that.